website/public/css/central-simmer.css

```diff
@@ -1,70 +1,747 @@
-.central-simmer .el-input {
-    --el-input-bg-color: var(--cs-primary);
-    --el-input-border-radius: 15px;
-    --el-input-border-color: rgba(255, 255, 255, 0);
-    --el-input-text-color: #ffffff;
+/*BASIC CONFIG STARTS*/
+
+/*Animation Snippets*/.central-simmer *, .central-simmer *:before, .central-simmer *:after, .central-simmer *:hover, .central-simmer *:active{
+    transition: width .09s, height .09s, transform .09s;
+    transition: .09s;
 }
 
-.central-simmer .el-input .el-input__inner {
-    padding: 0 20px;
+/*Element Hider*/.central-simmer .el-button+.el-button, .central-simmer .twitter-btn, .central-simmer .discord-btn, 
+.central-simmer .el-avatar, .central-simmer .el-alert, .central-simmer .logo-text[data-v-c970699f], 
+.central-simmer h6, .central-simmer .el-notification, .central-simmer .el-page-header__title, 
+.central-simmer .el-page-header__left .el-divider--vertical, .central-simmer .space-right[data-v-87359d7f]{
+    display: none !important;
 }
+/*Header in Light Scheme*/ .central-simmer .el-row[data-v-c970699f] {
```

Comment:
I think this is not a good idea to use a selector with `[data-v-c970699f]` as this is generated by `element-plus` or `vue` and can be broken at any time :/